authentik/stages/authenticator/models.py

```diff
@@ -85,6 +86,8 @@ class Device(models.Model):
 
     confirmed = models.BooleanField(default=True, help_text="Is this device ready for use?")
 
+    last_used = models.DateTimeField(null=True)
```

Comment:
We already have `last_t` for some authenticators.

I went with a new common field for all authenticators, that is only populated once the authenticator has been used in other contexts than setup.